<launch>
<arg name="camera" default="/camera"/>
<arg name="image" default="image_raw"/>
<arg name="fiducial_len" default="0.053"/>
<arg name="dictionary" default="7"/>
<node pkg="disp_pkg" name="disp_pkg" type="displacement3D.py" output="screen" respawn="false">
<param name="fiducial_len" value="$(arg fiducial_len)"/>
<param name="dictionary" value="$(arg dictionary)"/>
<remap from="camera" to="$(arg camera)/image_raw"/>
<remap from="camera_info" to="$(arg camera)/camera_info"/>
</node>
</launch>